recipes/prefect-ray/meta.yaml

```
@@ -0,0 +1,49 @@
+{% set name = "prefect-ray" %}
+{% set version = "0.2.0.post2" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 763effd3806e493a901c7542b8f3e25a92806222669b33cdb335c70a53050822
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv --no-deps"
+  # pickle5 only available for python 3.7
+  skip: true  # [py<37,osx]
+
+requirements:
+  host:
+    - pip
+    - python
+  run:
+    - pickle5 >=0.0.11  # [py<38]
+    - prefect >=2.3.0
+    - ray-core
+    - python
```

Please make this `noarch: python` and require `python  >=3.8` to be available on all conda platforms (there currently still are plenty python versions 3.8, 3.9, 3.10)) thx
```suggestion
  noarch: python

requirements:
  host:
    - pip
    - python  >=3.8
  run:
    - prefect >=2.3.0
    - ray-core
    - python >=3.8
```